docs/network/ble/ble_intro.md

```diff
@@ -25,6 +24,16 @@ NimBLE complies with Bluetooth Core Specification 5.0 which makes it an ideal wi
 
 The Bluetooth 5 is backward compatible with previous Bluetooth version 4.2 which is also supported by Apache Mynewt.
 
+### Bluetooth Mesh features
+
+Nimble supports Bluetooth Mesh which is great addition to IoT connectivity space and opens wide range of new possibilities for it.
```

Comment:
Maybe reword:
"NimBLE supports Bluetooth Mesh. Bluetooth Mesh opens a wide range of new possibilities for the IoT connectivity space.  NimBLE fully supports the following Bluetooth Mesh features:

Reply:
agree - will fix